v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/generator/registry.go

```diff
@@ -87,7 +93,9 @@ func newRegistry(req *plugin.CodeGeneratorRequest) (*registry, error) {
 // CodeGeneratorRequest with the registry.
 func (r *registry) Load(req *plugin.CodeGeneratorRequest) error {
```

Comment:
Unrelated to this diff, but it seems like `Load` should be private.

Reply:
Yup, good catch. Updated 👍